demo-deploy/README.md

```diff
@@ -0,0 +1,51 @@
+# Infrastructure Deployment Makefile
+
+This repository includes a `Makefile` to streamline the deployment, management, and teardown of AWS infrastructure using Terragrunt. The script ensures all prerequisites are met and simplifies executing commands for planning, applying, and destroying infrastructure across all modules in a specific environment.
+
+## Prerequisites
+
+Before using this script, ensure the following:
+
+1. **AWS CLI**:
+   - Install the AWS CLI.
+   - Configure it with your credentials.
+   - Ensure the `qed-it` AWS profile exists in `~/.aws/credentials`.
+
+2. **Terragrunt**:
+   - Install Terragrunt: [Install Instructions](https://terragrunt.gruntwork.io/docs/getting-started/install/).
+
+3. **Make**:
+   - Ensure `make` is installed on your system.
+
+4. **Repository Structure**:
+   - The script expects the `infra/terragrunt-aws-environments` directory to exist at the following location:
+     ```
+     /Users/arsenikalma/GIT/qed-it/zebra/demo-deploy/infra/terragrunt-aws-environments
```

Comment:
it does not make sense to create `/Users/arseni`. Consider relative to home `~/qed-it/zebra...`